README.md

```diff
@@ -68,7 +68,7 @@ Kili automl training does the following:
 * Finetunes the model on the input data.
 * Outputs the model loss.
 
-Here are the supported ML frameworks and the tasks they are used for.
+Here are the supported ML backends and the tasks they are used for:
```

Comment:
backends = Tensorflow, Pytorch, JAX
here we are talking about frameworks or model repositories

Reply:
```suggestion
Here are the supported ML repositories and the tasks they are used for:
```